core/src/main/scala/org/http4s/Message.scala

```diff
@@ -304,6 +304,12 @@ final case class Response(
 
   override def toString: String =
     s"""Response(status=${status.code}, headers=$headers)"""
+
+  /** Returns a list of cookies from the [[org.http4s.headers.Set-Cookie]]
+    * headers. Includes expired cookies, such as those that represent cookie
+    * deletion. */
+  def cookies: List[Cookie] =
```

Comment:
Laziness has tended to bite us.  I think if we did go that route, we'd more likely want to go the null init pattern we've used elsewhere (e.g., CI hash code, parsed header value).